docs/spectrum.rst

```diff
@@ -406,9 +406,6 @@ spectral axis, or 'spatial', which will collapse along all non-spectral axes.
     >>> spec.mean(axis='spatial')  # doctest: +FLOAT_CMP
     <Spectrum(flux=<Quantity [0.37273938, 0.53843905, 0.61351648, 0.57311623, 0.44339915,
                0.66084728, 0.45881921, 0.38715911, 0.39967185, 0.53257671] Jy> (shape=(10,), mean=0.49803 Jy); spectral_axis=<SpectralAxis
-       (observer to target:
-          radial_velocity=0.0 km / s
-          redshift=0.0)
```

Comment:
Why was this removed?  Is it an incidental thing you noticed at the time (no prob if so) or a result of this change somehow?

Reply:
I don't recall, I'll double check.

Reply:
I think this was due to creating the new collapsed `Spectrum` with `self.spectral_axis` rather than `self.wcs` now, and is actually probably more correct than the previously printed out `repr`, which was replacing `None` values for these quantities with `0` in the process of collapsing (implying information we actually don't have).